README.md

```diff
@@ -1,4 +1,63 @@
-# Logstash Plugin
+# Logstash Output Plugin
+
+---
+
+This plugin is now in maintenance mode. This is because the OpenSearch Project created a new Logstash output plugin 
```

Comment:
Please be a little clearer that we will continue to supply bug fixes and security fixes.